examples/terraform/aws/output.tf

```diff
@@ -73,7 +73,7 @@ output "kubeone_workers" {
     "${var.cluster_name}-${local.zoneA}" = {
       replicas = var.initial_machinedeployment_replicas
       providerSpec = {
-        sshPublicKeys   = [aws_key_pair.deployer.public_key]
+        sshPublicKeys   = var.worker_deploy_ssh_key ? [aws_key_pair.deployer.public_key] : []
```

Comment:
To avoid repeating the same thing, please define a local var (which will equal to this exact ternary expression) and use it in the rest of the config.

Reply:
Good point, changed. PTAL.